src/main/scala/io/iohk/ethereum/rlp/package.scala

```diff
@@ -43,4 +44,11 @@ package object rlp {
     * @throws RLPException if there is any error
     */
   def nextElementIndex(data: Array[Byte], pos: Int): Int = RLP.getItemBounds(data, pos).end + 1
+
+  trait RLPSerializable {
+    def toRLPEncodable: RLPEncodeable
+    def toBytes(implicit di: DummyImplicit): ByteString = ByteString(toBytes: Array[Byte])
+    def toBytes: Array[Byte] = encode(this.toRLPEncodable)
```

Comment:
I'm not sure about that, the idea of `MessageSerializable` not extending from `RLPSerializable` was to have `RLPSerializable` and `MessageSerializable` as decoupled as possible, so that in the future we could easily exchange `RLPSerializable` for another serializing algorithm. We also have some classes (such as `Block`) which are `RLPSerializable`s but not messages.